ui/packages/shared/profile/src/ProfileView.tsx

```diff
@@ -219,7 +219,7 @@ export const ProfileView = ({
                   </Button>
                 </div>
 
-                {callgraphEnabled ? (
+                {(callgraphEnabled as boolean) ? (
```

Comment:
`as boolean` should not be needed if the return type of the function is correct
```suggestion
                {callgraphEnabled ? (
```